upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -84,7 +87,8 @@ spec:
   strategy:
     type: RollingUpdate
     rollingUpdate:
-      maxUnavailable: 1
+      maxSurge: 10%
+      maxUnavailable: 0
```

Comment:
I've been investigating some DNS issues related to cluster rolling-updates, this seems related.

![thank you](https://media.giphy.com/media/1Z02vuppxP1Pa/giphy.gif)